pkg/services/serviceaccounts/manager/service.go

```diff
@@ -289,3 +292,38 @@ func validAPIKeyID(apiKeyID int64) error {
 	}
 	return nil
 }
+
+// hideApiKeysTabIfNoAPIKeysPresent is used to remove the apikeys tab from the admin tabs
+// if there are no apikeys, this makes it so that we only issue service account tokens instead
+func hideApiKeysTabIfNoAPIKeysPresent(l *log.ConcreteLogger, store *sqlstore.SQLStore, kvStore kvstore.KVStore) {
+	var err error
+	ctx := context.Background()
+	sess := store.GetSqlxSession()
+	orgs := make([]*org.OrgDTO, 0)
+	err = sess.Select(ctx, &orgs, "SELECT id, name FROM org")
+	if err != nil {
+		l.Error(fmt.Sprintf("unable to query for org for hiding api keys: %e", err))
+		return
+	}
+	if len(orgs) != 1 {
+		l.Info("do not want to hide apikeys for more than 1 org")
+		return
+	}
+
+	orgID := orgs[0].ID
+	apikeys := make([]*apikey.APIKey, 0)
+	if orgID != -1 {
```

Comment:
We use `-1` for orgId in special cases, like indicate that we want to query items across all orgs. But the org itself never has `-1` id I think.

Reply:
il use `-1` for globally hiding the API keys. pushing some changes soon

Reply:
made a const in the serviceaccounts package that is referred to as the org we save as hiding all the apikeys tabs